posts/2023-02-21-23.0.0.2-beta.adoc

```diff
@@ -0,0 +1,389 @@
++
+[.img_border_light]
+image::img/blog/ac_db_test_container_auth.png[Container authentication,width=50%,align="center"]
+
+
+* For applications that use application authentication, choose the "Application authentication" tab and fill in a valid user name and password for the database resource.
++
+[.img_border_light]
+image::img/blog/ac_db_test_app_auth.png[Application authentication,width=50%,align="center"]
+
+* If your application does not use a resource reference and you have not `enableContainerAuthForDirectLookups="true"` in the config element, then choose "No resource reference" tab and fill in a valid user name and password for the database resource.
++
+[.img_border_light]
+image::img/blog/ac_db_test_no_resource_ref.png[No Resource Reference,width=50%,align="center"]
+
+6. Click the 'Connection Test' button and results will be displayed at the bottom.  A successful Connection Test will look like this:
+
+[.img_border_light]
+image::img/blog/ac_db_test_successful_test.png[Successful connection test example,width=50%,align="center"]
+
+In addition to link:{url-prefix}/docs/latest/reference/feature/jdbc-4.3.html[Java Database Connectivity] you can also test connections to link:{url-prefix}/docs/latest/reference/feature/connectors-2.0.html[Jakarta Connectors], link:{url-prefix}/docs/latest/reference/feature/messaging-3.0.html[Jakarta Messaging] and link:{url-prefix}/docs/latest/reference/feature/cloudant-1.0.html[Cloudant Integration] resources. 
+
+For more information regarding administering Liberty using a GUI, refer to the link:{url-prefix}/docs/latest/admin-center.html[Manage Open Liberty with Admin Center] documentation.
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/23987
+// Contact/Reviewer: njr-11,ryan-storey
+// // // // // // // // 
+[#data]
+== A very early preview of Jakarta Data
+Jakarta Data is a new Jakarta EE specification that aims to standardize the popular Data Repository pattern across relational and NoSQL databases. As the specification is further developed and Jakarta Data providers become available, it is expected that the Jakarta Data specification will standardize a way to plug in your favorite providers that are currently dominant in the industry, including JNoSQL, to be used in Open Liberty. For now, Open Liberty is making available as beta a test implementation of Jakarta Data that we are using to experiment with proposed specification features so that developers can try out these features and provide feedback to influence the specification as it is being developed. The test implementation currently works with relational databases and operates by redirecting repository operations to the built-in Jakarta Persistence provider. It simulates the entirety of the Jakarta Data beta release, plus some additional proposed features that are being considered.

```

Comment:
```suggestion
Jakarta Data is a new Jakarta EE specification that aims to standardize the popular Data Repository pattern across relational and NoSQL databases. As the specification is further developed and Jakarta Data providers become available,  the Jakarta Data specification is expected to standardize a way to plug in your favorite providers that are currently dominant in the industry, including JNoSQL, to be used in Open Liberty. 

For now, Open Liberty is using a test implementation of Jakarta Data to experiment with proposed specification features. This implementation is now available in our beta release so that developers can try out these features and provide feedback to influence the specification as it is being developed. The test implementation currently works with relational databases and operates by redirecting repository operations to the built-in Jakarta Persistence provider. It simulates the entirety of the Jakarta Data beta release, plus some additional proposed features that are being considered.

```